install.sh

```diff
@@ -100,16 +101,21 @@ function curl () {
 }
 
 case "${ARCH}" in
-"i686"|"x86_64"|"armv7l"|"aarch64")
+"i686")
+  echo_error "Installs on i686 devices are no longer supported by Chromebrew. :/"
+  exit 1
+  ;;
```

Comment:
So this is the official end of i686 support?

Reply:
It's time to do it :)

Reply:
Of course, if i686 support is holding us back in other areas than feel free to get rid of it, but if its only because we don't want to maintain the installer support than thats a non-issue.

Reply:
How long do you think it will be before we get a true `aarch64` architecture? I'd hate to see my nice and refactored install.sh have a bunch of temporary `armv7` on `aarch64` code for too long.

Reply:
> How long do you think it will be before we get a true `aarch64` architecture? I'd hate to see my nice and refactored install.sh have a bunch of temporary `armv7` on `aarch64` code for too long.

Honestly I haven't even started that yet. I just got our `armv7l` userspace working on the official `aarch64` userspace.

Bootstrapping `aarch64` means building glibc and GCC from scratch, or ideally enabling the official `dev_install` first on a current `aarch64` Chromebook... And I don't have one of those.

Reply:
I added back `i686`...